src/Sarif.WorkItems/SarifWorkItemsExtensions.cs

```diff
@@ -75,6 +77,35 @@ public static string CreateWorkItemTitle(this Run run)
                     (locationName == null ? "" : " (in " + locationName + ")");
         }
 
+        public static Dictionary<string, int> FetchToolResultCounts(this SarifLog log)
+        {
+            if (log == null) { throw new NullReferenceException(); }
```

Comment:
>NullReferenceException [](start = 41, length = 22)

Throw ArgumentNullException here

Reply:
Changed the identical check for title as well.

---
In reply to: [394695607](https://github.com/microsoft/sarif-sdk/pull/1817#discussion_r394695607) [](ancestors = 394695607)